src/tests/unit/tests/DetailsView/components/assessment-instance-edit-and-remove-control.test.tsx

```diff
@@ -61,7 +61,7 @@ describe('AssessmentInstanceRemoveButton', () => {
 });
 
 class TestableAssessmentInstanceRemoveButton extends AssessmentInstanceEditAndRemoveControl {
-    public getOnRemoveButtonClicked() {
+    public getOnRemoveButtonClicked(): (event?: React.MouseEvent<HTMLElement | HTMLAnchorElement | HTMLButtonElement, MouseEvent>) => void {
```

Comment:
I see the `, MouseEvent` here again!, It's at the end of the templates but that `MouseEvent` is not part of the union!

Reply:
I guess thats how a `React.MouseEvent` is defined and thats why vscode suggests me those. But fair point, we dont need them, so changed it.